src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx

```diff
@@ -20,16 +20,55 @@ If your users are on our [original user model](#user-models), you may have the a
 
 ## How paying accounts switch to new pricing [#transition-paid]
 
-If you're a paying account on our original pricing plan, you may only switch to the new pricing if one of these is true: 
+If you're a paying account on our [original pricing plan](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models), you may only switch to New Relic One pricing if one of these is true: 
 
 * You're on month-to-month billing.
 * You're within 60 days of your subscription renewal. 
 
-If one of those is true, you may have access to a self-serve pricing migration in the UI. To find that: from the account dropdown, click **Manage your plan** and then click **Plans and pricing** ([direct link](https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJhY2NvdW50SWQiOjI0NDEyODYsIm5lcmRsZXRJZCI6ImNvbW1lcmNlLW5lcmRsZXRzLm5yLXNob3AtZW50cnkiLCJwYXRoIjoiL3BsYW5zLWFuZC1wcmljaW5nIiwibWlncmF0aW9uRWxpZ2libGUiOnRydWUsImFjY291bnROYW1lIjoiU3RyYWRvcyBMYWJzIn0=&platform[accountId]=)). If you don't see that UI, contact your account representative, if you have one. 
+If one of those is true, you may have access to a self-serve pricing migration in the UI. To use that UI: 
 
-Have more questions? You can ask in the [Explorers Hub](https://discuss.newrelic.com/). 
+1. To find the UI option: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown), click **Manage your plan** and then click **Plans and pricing** ([direct link](https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJhY2NvdW50SWQiOjI0NDEyODYsIm5lcmRsZXRJZCI6ImNvbW1lcmNlLW5lcmRsZXRzLm5yLXNob3AtZW50cnkiLCJwYXRoIjoiL3BsYW5zLWFuZC1wcmljaW5nIiwibWlncmF0aW9uRWxpZ2libGUiOnRydWUsImFjY291bnROYW1lIjoiU3RyYWRvcyBMYWJzIn0=&platform[accountId]=)). If you don't see an **Unlock full platform access** UI, contact your account representative or ask a question in the [Explorers Hub](https://discuss.newrelic.com/). 
```

Comment:
This line has a direct link embedded. Is that intentional? Is that a generic " link that anyone can go to, or should you remove the ()[] and put the period after "Plans and pricing"?

Reply:
it's supposed to work for everyone. is what they told me anyway.